attributes.json

```diff
@@ -0,0 +1,8 @@
+{
+  "dd-agent-builder": {
+      "omnibus-ruby_branch": "datadog-5.5.0"
+  },
+  "omnibus": {
+     "build_user_password": "DDBuildR0cks!"
```

Comment:
➕ for storing this in an env var on gitlab, if the password is actually used.

Could we pass/export the env var so that the `chef-client` call can pick it up? (and then use the env var as the default value of `build_user_password` in `attributes/default.rb`?)